src/js/providers/html5.js

```diff
@@ -502,14 +526,19 @@ define([
                 });
             }
 
-            if (_canSeek) {
+            if (_getSeekableEnd() >= seekPos) {
+                // We can seek if the end seekable range is at or past the seek position
+                _loadInProgress = true;
+            }
+
+            if (_loadInProgress) {
```

Comment:
I tried to handle maximum seekable range, but it looks like this is not necessary. I'll remove this and change the variable name back to canSeek, since loadInProgress is misleading.